css/styles.css

```diff
@@ -480,27 +481,32 @@ textarea {
 }
 
 @media (min-width: 780px) {
-  header {
+  .nav-menu {
     position: fixed;
+    z-index: 100;
+    height: 104px;
     display: flex;
-    width: 100%;
+    width: 1393px;
   }
 
   #icon-menu {
     display: none;
   }
 
-  .menu_desk {
+  .first_section {
     display: flex;
-    margin-right: 164px;
+    width: 100%;
   }
 
-  .mylogo {
-    margin-left: 155px;
+  .menu_desk {
```

Comment:
I am not really sure if this problem is new because the wrong line of code could not be found in your current CSS commit but make sure your items are justified to the center based on the Figma design:
putting `display:flex` and `justify-content:center` if flex-direction is row or  `align-items:center`  if  flex direction is column in the parent container, might fix this issue.

![image](https://user-images.githubusercontent.com/43564324/165986714-cadcf927-0b25-42f0-b7cc-5850f2654616.png)